sources/packages/web/src/components/students/applicationTracker/Completed.vue

```diff
@@ -53,6 +53,70 @@
     :background-color="hasDisbursementEvent ? undefined : 'success-bg'"
     content="StudentAid BC has determined an outcome with 1 or more of your requested change. Please review your new assessment in the table below."
   />
+  <!-- Student application change request - in progress with student -->
```

Comment:
It is not a must, but I would recommend to have the application tracker screenshots for following remaining scenarios as well.

 - added a warning card showing the request is pending SABC approval if the student approves the change request.
 - added a student declined card if the student declines the change request.
 - added a ministry approved card if the ministry approves the change request.
 - added a ministry declined card if the ministry declines the change request.

Reply:
Thanks @dheepak-aot. Added the screenshots.